L2 Understading electric vehicless/Different types of motors used in Electric Vehicles.md

```diff
@@ -1,66 +1,67 @@
 [Content Suggestions: This content can be broken down into two chapters, and some more content like https://www.youtube.com/watch?v=CWulQ1ZSE3c or
 https://youtu.be/bCEiOnuODac?feature=shared can be added in the future]: #
```

Comment:
@Gurudayal19 Do you see any scope/need of inclusion of these two videos?

Reply:
Yes, the first video covers the idea of motor as a very basic concept, and the second video ramps it up. Thus, both these videos are essential and as per the latest [EV101 content](https://www.pupilfirst.school/targets/10106) on LMS, these videos have been moved to [EV201](https://www.pupilfirst.school/targets/11461).

Answering your `Content Suggestion`, if these were to be converted to text-based content in EV101, then it would be too much of information there, and we have anyway covered it in detail in EV201.